Tasks/MavenV2/maventask.ts

```diff
@@ -423,6 +423,7 @@ function publishCodeCoverage(isCodeCoverageOpted: boolean): Q.Promise<boolean> {
             }
             mvnReport.line(mavenOptions);
             mvnReport.arg("verify");
+            mvnReport.arg("-Dmaven.test.skip=true");
```

Comment:
Nit: Could you please add some comments regarding the `-Dmaven.test.skip` argument with the link to online documentation?

Reply:
Added, thanks!